Hyper/Meshes/Mesh.cs

```diff
@@ -20,18 +23,27 @@ internal class Mesh : IDisposable
 
     protected int NumberOfVertices;
 
+    private bool createdVertexArrayObject;
+
     public Vertex[] Vertices { get; protected set; }
 
     public Mesh(Vertex[] vertices, Vector3 position)
     {
+        createdVertexArrayObject = false;
```

Comment:
once this variable is set to true, it can never get back to false again - do we need it in this case?